client/js/lounge.js

```diff
@@ -585,7 +586,20 @@ $(function() {
 			// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Iterating_over_own_properties_only
 			for (let key in params) {
 				if (params.hasOwnProperty(key)) {
-					const value = params[key];
+					let value = params[key];
+
+					if (key === "join") {
+						let channels = value.split(",");
+						channels = _.map(channels, (c) => {
```

Comment:
Use a native method instead of bringing lodash here (we don't use it on the client).

Reply:
I've addressed this and squashed the commit.